### ISSUE TITLE
XSL Transformation for HTML-Output to show sigle only when speaker changes

### ISSUE THREAD

**Issue by wuddel** [EXMARaLDA/exmaralda#403]
So far every line has a speaker sigle, as in:

> SPK1: Hello.
> SPK2: Hello there.
> SPK2: How are you?
> SPK2: Everything fine?
> SPK1: Yes.

it would be easier readable with:

> SPK1: Hello.
> SPK2: Hello there.
> ______ How are you?
> ______ Everything fine?
> SPK1: Yes.

I suggest the following changes in the XSLT(s) in order to achieve this:

```
<td class="speaker">
  <xsl:choose>
    <xsl:when test="position() = 1 or preceding-sibling::contribution[1]/@speaker-reference != @speaker-reference">
      <xsl:value-of select="@speaker-reference"/>
    </xsl:when>
    <xsl:otherwise>
    </xsl:otherwise>
  </xsl:choose>
</td>
```

**Comment by berndmoos** [comment]
cf. #183

**Comment by berndmoos** [comment]
Closed via #413
